datadog/provider.go

```diff
@@ -111,7 +110,7 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 
 	c := cleanhttp.DefaultClient()
 	c.Transport = logging.NewTransport("Datadog", c.Transport)
-	communityClient.ExtraHeader["User-Agent"] = getUserAgent("")
+	communityClient.ExtraHeader["User-Agent"] = getUserAgent("go-datadog-api")
```

Comment:
maybe we could manually add the version since you are pinning it in `GNUmakefile`